front/src/reducers/user/userReducer.spec.ts

```diff
@@ -63,6 +64,17 @@ describe('userReducer', () => {
     expect(userState).toEqual(userInitialState);
   });
 
+  it('should handle setUserRoles', () => {
+    const store = createStore(userInitialState);
+    const action = setUserRoles({ userRoles: ['Stdcm'] });
+    store.dispatch(action);
```

Comment:
You can directly dispatch
```suggestion
    store.dispatch(setUserRoles(['Stdcm']));
```